Duke.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Duke {
+
+    private static int Count = 0;
+    private static Task[] List = new Task[10]; //creating list of Task for inputs
+
+    //add task object into list
+    public static void addList(Task s){
+        List[Count] = s;
+    }
+
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
```

Comment:
Hi. The welcome message could be extracted into one method to make the main method shorter and more readable!